src/server/common/model/answer/on-off-farm.test.js

```diff
@@ -73,6 +73,17 @@ describe('#OnOffFarm.fromState', () => {
   })
 })
 
+describe('OnOffFarm.errorMessages', () => {
```

Comment:
If we just use the static method from answer model we might want to also unit test it in the model only?

Reply:
see above